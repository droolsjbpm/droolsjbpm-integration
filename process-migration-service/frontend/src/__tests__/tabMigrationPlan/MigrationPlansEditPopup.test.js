import renderer from "react-test-renderer";
import React from "react";
import MigrationPlansEditPopup from "../../component/tabMigrationPlan/MigrationPlansEditPopup";

test("MigrationPlansEditPopup renders correctly using snapshot", () => {
  const myMock = jest.fn();
  const tree = renderer
    .create(
      <MigrationPlansEditPopup
        title="Import Migration Plan"
        actionName="Import Plan"
        retrieveAllPlans={myMock}
        updatePlan={myMock}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
