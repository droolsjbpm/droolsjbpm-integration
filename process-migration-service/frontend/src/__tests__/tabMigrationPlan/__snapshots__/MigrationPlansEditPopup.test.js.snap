// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MigrationPlansEditPopup renders correctly using snapshot 1`] = `
<span>
  <button
    className="btn btn-default"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Import Plan
  </button>
</span>
`;
